frontend/src/app/shared/components/principal/principal-renderer.service.ts

```diff
@@ -170,6 +191,11 @@ export class PrincipalRendererService {
     return id ? this.apiV3Service.users.id(id).avatar.toString() : null;
   }
 
+  private userHoverCardUrl(principal:PrincipalLike|IPrincipal):string|null {
+    const id = principal.id || idFromLink(hrefFromPrincipal(principal));
+    return id ? `/users/${id}/hover_card` : null;
```

Comment:
This will fail in case of relative URL root installations. Please use the `PathHelperService` to generate static links, they are correctly using the appBase/releative URL root path